versioned/storage/jdbc/src/main/java/org/projectnessie/versioned/storage/jdbc/AbstractJdbcPersist.java

```diff
@@ -508,6 +508,13 @@ private boolean[] upsertObjs(
               if (updated[i] == 1) {
```

Comment:
tangential: what about `updated[i] > 1`?.. Should we treat it as an unexpected error too?

Reply:
Since each statement we do is a simple row insert, I guess that could never happen. But OK to add a check.